scripts/update_and_rebuild_petsc.sh

```diff
@@ -118,6 +118,7 @@ if [ -z "$go_fast" ]; then
       --with-cxx-dialect=C++11 \
       --with-fortran-bindings=0 \
       --with-sowing=0 \
+      --with-64-bit-indices \
```

Comment:
Do we not want to capture this differently? Like, add this only if the custom arg `--without-64-bit-indices` is not set?

I'm thinking about how to test 32 bit indices on CIVET if we're using these scripts.

Reply:
Maybe

```
--with-64-bit-indices --with-64-bit-indices=0
```

will work? I don't know.

Reply:
Yes, this will work. If users want to use 32-bit indices, they could simply pass --with-64-bit-indices=0 to override what we have.

PETSc-alt will use 32-bit always.